.travis.yml

```diff
@@ -14,6 +14,10 @@ compiler:
 env:
   global:
     - CMAKE_OPTIONS="-DOPTION_BUILD_EXAMPLES=On"
+    - PATH="/usr/local/opt/qt/bin:$PATH"
+    - PATH="/usr/local/opt/glbinding:$PATH"
+    - PATH="/usr/local/opt/globjects:$PATH"
```

Comment:
I would put these all into one line, otherwise the list format suggests that three separate variables are being set

Reply:
I'd do that, too.. actually I was confused about the ":", I expected a ";", so I just copied the line to make sure I won't make syntax mistakes :D I'm not used to linux nor macOS